applications/dirgraph_visualization/dirgraph_utils.py

```diff
@@ -0,0 +1,647 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This code will create a directed graph by reading a .json file.
+"""
+import dirgraph_wire as ntwku
+import dirgraph_steady_bc
+import dirgraph_connections
+import numpy as np
+import pandas as pd
+import json
+import pdb
+import sys
+import re
+import os
+import copy
+
+try:
+    from tqdm import tqdm
+except ImportError:
+    pass
+
+try:
+    import matplotlib.pyplot as plt  # only needed if you want to visualize the 0d model as a directed graph
+except ImportError:
+    print(
+        "\nmatplotlib.pyplot not found. matplotlib.pyplot is needed only if you want to visualize your 0d model as a directed graph.")
```

Comment:
Change this and the next few error messages to "___ is needed for visualizing the 0D model."